clients/browser/index.html

```diff
@@ -427,11 +427,11 @@
 
             var el = document.createElement('div');
             var date = new Date(head.timestamp * 1000);
-            var html = `<div><b>${date}</b> hash=<hash>${$.blockchain.headHash.toBase64()}</hash>, `
+            var html = `<div><b>${date}</b> hash=<hash>${$.blockchain.headHash}</hash>, `
                 + `miner=<hash>${head.minerAddr}</hash>, difficulty=${head.difficulty}</div>`;
 
             for (var tx of head.transactions) {
-                var senderAddr = 'TODO'; //await tx.getSenderAddr();
+                var senderAddr = await tx.getSenderAddr();
```

Comment:
Avoid using async/await here. Since order of transactions is not really important (yet), one simple solution would be to just create a promise for each transaction, wait for all of them to finish with `Promise.all()` and then join the resulting strings.